src/main/java/com/mongodb/hibernate/internal/translate/AbstractMqlTranslator.java

```diff
@@ -217,11 +223,35 @@ public void visitColumnWriteFragment(ColumnWriteFragment columnWriteFragment) {
     }
 
     // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    // Table Mutation: deletion
+
+    @Override
+    public void visitStandardTableDelete(TableDeleteStandard tableDeleteStandard) {
```

Comment:
We talked about `TableDeleteStandard.getMutationComment` during the code walkthrough, but I don't remember what we decided about it.

If we silently ignore it now here, we may never remember that we are not handling it properly. We also can't throw `FeatureNotSupportedException` if this comment is present, because it is always present, as it is generated by Hibernate ORM.

The best way forward appears to be creating a ticket for supporting comments, which we know may be given to us either via `TableDeleteStandard.getMutationComment`, or via `QueryOptions.getComment`.

Reply:
I created a ticket at https://jira.mongodb.org/browse/HIBERNATE-53